### ISSUE TITLE
Cannot resolve module in 'react-router' project

### ISSUE THREAD

**Issue by jonaswindey** [yahoo/flux-examples#95]
When running npm run build in /react-router (after doing npm install in the root folder), I get:

```
> flux-example-routing@0.0.0 build /Users/jonas/Development/github/flux-examples/react-router
> ../node_modules/.bin/webpack

Hash: 109e566318d685cc4e36
Version: webpack 1.5.3
Time: 630ms
    Asset    Size  Chunks             Chunk Names
client.js  612673       0  [emitted]  main
   [0] ./client.js 1052 {0} [built] [1 error]
   [1] ./app.js 458 {0} [built] [1 error]
   [4] ./stores/ApplicationStore.js 1411 {0} [built] [1 error]
   [5] ./stores/TimeStore.js 860 {0} [built] [1 error]
   [6] ./stores/PageStore.js 845 {0} [built] [1 error]
 [144] ./actions/updateTime.js 254 {0} [built]
    + 156 hidden modules

ERROR in ./client.js
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router
 @ ./client.js 12:13-36

ERROR in ./app.js
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router
 @ ./app.js 6:18-41

ERROR in ./stores/ApplicationStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/ApplicationStore.js 6:18-59

ERROR in ./stores/PageStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/PageStore.js 6:18-59

ERROR in ./stores/TimeStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/TimeStore.js 6:18-59

ERROR in ./components/Routes.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Routes.jsx 2:12-35 3:19-42

ERROR in ./components/Application.jsx
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Application.jsx 10:17-40

ERROR in ./components/Application.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Application.jsx 11:19-42

ERROR in ./components/Nav.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Nav.jsx 7:11-34 8:17-40

ERROR in ./components/Timestamp.jsx
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Timestamp.jsx 9:17-40
```

**Comment by jedireza** [comment]
Thanks for creating an issue. The `react-router` example needs to be updated.

**Comment by jedireza** [comment]
Fixed in #96
